CMakeLists.txt

```diff
@@ -242,7 +242,7 @@ set (
 set (
     ADUC_PLATFORM_LAYER
     "linux"
-    CACHE STRING "The platform layer to use. Options: simulator linux")
```

Comment:
is ADUC_PLATFORM_LAYER used anymore?  (It likely will be for WIOT) -- it seems like you've removed some of the checks on ADUC_PLATFORM_LAYER below and now assume linux?

Reply:
e.g. src/agent/CMakeLists.txt

Reply:
Added the platform check back.